charts/elasticsearch/templates/exporter/es-exporter-deployment.yaml

```diff
@@ -56,12 +56,12 @@ spec:
           image: {{ template "exporter.image" . }}
           imagePullPolicy: {{ .Values.images.exporter.pullPolicy }}
           command: ["elasticsearch_exporter",
-                    "-es.uri=http://{{ .Release.Name }}-elasticsearch:{{ .Values.common.ports.http }}",
-                    "-es.all={{ .Values.exporter.es.all }}",
-                    "-es.indices={{ .Values.exporter.es.indices }}",
```

Comment:
I guess we don't use these values, but we could do

```
{{if .Values.exporter.es.indices }}
                     "-es.indices=", 
{{ endif }}
```

(or whatever the text/template syntax is for ifs). Do we need to?